qemu/tests/cfg/numa_prealloc_threads.cfg

```diff
@@ -0,0 +1,20 @@
+- numa_prealloc_threads:
+    no RHEL.6 RHEL.7 RHEL.8
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    required_qemu = [7.2,)
+    type = numa_prealloc_threads
+    virt_test_type = qemu
+    login_timeout = 1000
```

Comment:
Not used.

Reply:
Included, because of taskset is being used, it could take more time to boot the vm